rotkehlchen/chain/ethereum/modules/eth2/accountant.py

```diff
@@ -0,0 +1,26 @@
+from typing import TYPE_CHECKING
+
+from rotkehlchen.accounting.structures.base import get_tx_event_type_identifier
+from rotkehlchen.accounting.structures.types import HistoryEventSubType, HistoryEventType
+from rotkehlchen.chain.evm.accounting.interfaces import ModuleAccountantInterface
+from rotkehlchen.chain.evm.accounting.structures import TxEventSettings
+
+from .constants import CPT_ETH2
+
+if TYPE_CHECKING:
+    from rotkehlchen.accounting.pot import AccountingPot
+
+
+class Eth2Accountant(ModuleAccountantInterface):
+    def event_settings(self, pot: 'AccountingPot') -> dict[str, TxEventSettings]:  # pylint: disable=unused-argument  # noqa: E501
+        """Being defined at function call time is fine since this function is called only once"""
+        return {
+            get_tx_event_type_identifier(HistoryEventType.STAKING, HistoryEventSubType.DEPOSIT_ASSET, CPT_ETH2): TxEventSettings(  # noqa: E501
+                taxable=False,
+                count_entire_amount_spend=False,
+                count_cost_basis_pnl=False,
```

Comment:
This should be true. This is how rotki will know where did the `ETH2` asset comes from